# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../dash/tempfile-fix.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- dash-0.5.1/src/mkbuiltins~	2003-02-15 05:43:14.000000000 +0100
+++ dash-0.5.1/src/mkbuiltins	2005-01-04 16:59:34.178730376 +0100
@@ -38,8 +38,8 @@
 #	@(#)mkbuiltins	8.2 (Berkeley) 5/4/95
 
 trap 'rm -f $temp $temp2' EXIT
-temp=$(tempfile)
-temp2=$(tempfile)
+temp=$(mktemp)
+temp2=$(mktemp)
 
 shell=$1
 builtins=$2
